Cargo.toml

```diff
@@ -10,3 +13,11 @@ panic = "abort"
 
 [profile.release]
 panic = "abort"
+
+[profile.min-size]
+codegen-units = 1
+inherits = "release"
+lto = true
+opt-level = "s"
+panic = "abort"
```

Comment:
Is this needed if we inherit from `release` ? Assuming my understanding of `inherits` is correct

Reply:
I don't think it's necessary, but if we ever change `panic` in `release` we have to keep in mind to change it in the new profile too. Since it directly influences the size (https://github.com/johnthagen/min-sized-rust#abort-on-panic) I thought it was good to keep it here just in case. But I can remove it if you want.